app/admin_api_keys.py

```diff
@@ -28,23 +28,25 @@
 API_KEY_LENGTH = 16
 KEYS_PER_PAGE = 50
 
+# Authorization params list
```

Comment:
Nit: I think you can delete this comment. It's not needed since it's redundant with the name of the constant itself.

Reply:
:+1: